Oqtane.Server/Extensions/StringExtensions.cs

```diff
@@ -27,5 +28,10 @@ public static string GetMimeType(this string fileName)
 
             return contentType;
         }
+
+        public static string GetMimeType(this File file)
```

Comment:
I suggest to move this into `FileExtensions` also you could use inline lambda expression body